pkg/injector/injectors_factory.go

```diff
@@ -0,0 +1,50 @@
+package injector
```

Comment:
Thoughts on naming scheme for encoders and evasions being similar to this?
* `evasion.go` -> `evasions_factory.go`
* `encoder.go` -> `encoders_factory.go`